scripts/pre_commit_linter.py

```diff
@@ -1006,6 +1005,9 @@ def _check_docstrings(all_files):
         'Single line docstring should not span two lines. '
         'If line length exceeds 80 characters, '
         'convert the single line docstring to a multiline docstring.')
+    previous_line_message = (
+        'There should not be more than one new line before the end '
```

Comment:
This message seems to be a bit confusing. @kevinlee12 what do you think?

Reply:
It seems okay, but I can see where it can be confusing. I think correcting it to: `There should not be any empty lines before the end of the multi-line docstring.`

Reply:
That sounds perfect, Kevin :) @lilithxxx, can you please go ahead and make the necessary changes? Thanks!

Reply:
Done @apb7 !